.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+
+const prettierOptions = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, '.prettierrc'), 'utf8'),
+);
+
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: ['airbnb', 'prettier', 'prettier/react'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+  plugins: ['react', 'prettier', '@typescript-eslint'],
+  rules: {
```

Comment:
co Ty na to żeby dodać te dwie reguły ? 
`'newline-before-return', 'newline-after-var',`

Reply:
Widzę, że te reguły są `deprecated`, ale można obie zastąpić tą regułą:
https://eslint.org/docs/rules/padding-line-between-statements
Zaraz coś dodam.